src/main/java/uk/gov/hmcts/darts/common/entity/Media.java

```diff
@@ -62,4 +62,6 @@ public class Media {
     @Column(name = "i_version_label")
     private Short version;
 
+    @Column(name = "i_storage_location", length = 64)
```

Comment:
I think just call this

```suggestion
    @Column(name = "storage_location", length = 64)
```

Also where did the length come from?